src/index.html

```diff
@@ -1,63 +1,85 @@
-          <div class="left-header">
-            <img
-              src="./assets/daily.svg"
-              class="daily-logo"
-              alt="Daily logo reads Daily"
-            />
-            <strong>
-              <p class="header-title">Daily demo</p>
-            </strong>
-          </div>
-          <div class="right-header">
-            <a
-              href="https://docs.daily.co/reference"
-              target="_blank"
-              class="link"
-            >
-              <button class="button docs-button white-button">
-                API docs
-                <img
-                  class="arrow-icon"
-                  src="./assets/new-tab-icon.png"
-                  alt="Arrow icon points up and to the right"
-                  aria-hidden="true"
-                />
-              </button>
-            </a>
-            <span class="divider"></span>
-            <a href="https://github.com/daily-demos/boilerplate-vanilla-js/" target="_blank">
+      <div class="header-container">
+        <div class="left-header">
+          <img
+            src="./assets/daily.svg"
+            class="daily-logo"
+            alt="Daily logo reads Daily"
+          />
+          <strong>
+            <p class="header-title">Daily Prejoin Presence Demo</p>
+          </strong>
+        </div>
+        <div class="right-header">
+          <a
+            href="https://docs.daily.co/reference"
+            target="_blank"
+            class="link"
+          >
+            <button class="button docs-button white-button">
+              API docs
               <img
-                class="octocat"
-                src="./assets/github.png"
-                alt="GitHub logo Octocat"
+                class="arrow-icon"
+                src="./assets/new-tab-icon.png"
+                alt="Arrow icon points up and to the right"
+                aria-hidden="true"
               />
-            </a>
+            </button>
+          </a>
+          <span class="divider"></span>
+          <a
+            href="https://github.com/daily-demos/boilerplate-vanilla-js/"
+            target="_blank"
+          >
+            <img
+              class="octocat"
+              src="./assets/github.png"
+              alt="GitHub logo Octocat"
+            />
+          </a>
+        </div>
+      </div>
+    </header>
+    <noscript>
+      This application requires JavaScript to run. Please enable JavaScript in
+      your browser.
+    </noscript>
+    <div id="container">
+      <div id="entry">
+        <button id="createRoom">Create Room</button>
+      </div>
+      <div id="call" class="hidden">
+        <div id="callFrame"></div>
+        <div id="sidebar">
+          <div id="invite">Invite URL:
+            <br /><a id="inviteURL" href=""></a></div>
+          <hr />
+          <div id="presence" class="hidden">
+            The following participants are already in the call:
+            <ul id="participants"></ul>

```

Comment:
When I join the call and this list gets removed, I was a little surprised. Joining the call feels like the chance to confirm the prejoin list was correct so I think it would be nice to keep it (maybe replace it with the `participants()` list and update the text above to "The following participants are in this call" or something.

Aside: the text above should be in a text tag (e.g. a `<p>` tag). You don't typically want the element wrapping text to be a div (not descriptive enough).

Reply:
I was hesitant to keep the list because this demo does not feature live update of the list, so if people join or leave the call once the user is in, the list will appear inaccurate (even though it was actually correct at the time of joining). 

But since I have now added a notice that they need to refresh the page to see an updated list, I think it might be more intuitive to keep it. I'll try that!

Reply:
Good point about keeping it updated. Will leave to you but I think it would be a nice addition.